hibernate-core/src/main/java/org/hibernate/dialect/CockroachDB201Dialect.java

```diff
@@ -0,0 +1,17 @@
+package org.hibernate.dialect;
+
+import org.hibernate.dialect.function.NoArgSQLFunction;
+import org.hibernate.type.StandardBasicTypes;
+
+/**
+ * An SQL dialect for CockroachDB 20.1 and later. Adds support for a few date/time functions.
+ */
+public class CockroachDB201Dialect extends CockroachDB192Dialect {
+	public CockroachDB201Dialect() {
+		super();
```

Comment:
technically the default invocation of super can be omitted. it is only necessary when parameter(s) needed.

Reply:
I noticed that other dialects include the explicit `super()` call. e.g. `PostgreSQL94Dialect`.